products/numpy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NumPy
+permalink: /numpy
+category: framework
+releasePolicyLink: https://numpy.org/neps/nep-0029-deprecation_policy.html
+versionCommand: python -c "import numpy; print(numpy.__version__)"
+auto:
+-   git: https://github.com/numpy/numpy
```

Comment:
This is invalid. 

- auto should be a list.
- git and regex are keys on the same dict/map.
- git needs to point to a valid git repository (so end in `.git`).

You can look at python.md for reference

Reply:
I've removed the leading `-` and added the `.git` extension. I think that should change the type to a list.